activelearning/acquisition/acquisition.py

```diff
@@ -24,19 +24,19 @@ def __init__(
         dataset_handler: DatasetHandler,
         device: Union[str, torch.device],
         float_precision: Union[int, torch.dtype],
-        maximize=False,
     ) -> None:
         self.dataset_handler = dataset_handler
         self.device = device
         self.float = set_float_precision(float_precision)
-        self.target_factor = 1 if maximize else -1
-        self.maximize = maximize
         self.surrogate_model = surrogate_model
 
     def __call__(self, candidate_set: torch.Tensor) -> torch.Tensor:
+        # sampler (GFlowNet) expects minimization problem, but acquisition function return maximization problem -> always invert
+        # all samplers are fixed to minimize
+        # TODO: remove -1; this will later be implemented directly in the gflownet environment (currently it always asumes negative values i.e. minimizing values)
```

Comment:
I would write these comments as docstring (with """):
```suggestion
"""
Evaluates the acquisition function on a set of candidates.

Note that the sampler (GFlowNet) operates as in a minimization problem, but the acquisition function is to be maximized. Therefore, the outputs are inverted.

TODO: remove -1; this will later be implemented directly in the gflownet environment (currently it always assumes negative values i.e. minimizing values)
```

Reply:
I think the indentation will be wrong in my suggestion.